utils/sv_export_png.py

```diff
@@ -39,7 +39,7 @@ def png_pack(png_tag, data):
                 chunk_head +
                 struct.pack("!I", 0xFFFFFFFF & zlib.crc32(chunk_head)))
 
-    t = color_type[type]
+    t = color_type[mode]
```

Comment:
to improve code readibility i think i will turn this line and others to something:

```python
def png_header():
     return b'\x89PNG\r\n\x1a\n'
```

```diff
+ return b''.join([
- b'\x89PNG\r\n\x1a\n',
+ png_header(),
```